AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -57,6 +57,7 @@ gaugau.futabanet.jp,newscientist.com,jiji.com,edp24.co.uk,technologyreview.com,n
 !
 ! SECTION: Subscriptions - Regular rules
 !
+boulderreportinglab.org##.newspack-popup-container:has(> div[class] > div[class] > div[class] .wp-block-newspack-newsletters-subscribe)
```

Comment:
I think hiding of `.newspack-popup-container` is ok. It will also hide a large donation  suggestion, but they have a button at the top. And both elements are an annoyance.

```suggestion
boulderreportinglab.org##.newspack-popup-container
```

You may also add the same rule here
https://github.com/AdguardTeam/AdguardFilters/blob/37bd44a103c0bc7afd225bcfe59655936cae1d1c/AnnoyancesFilter/Widgets/sections/widgets.txt#L2431-L2432